public/static/locales/en/dashboard.json

```diff
@@ -35,6 +35,7 @@
     "origination": "{{origination}} {{user}} on {{date}}",
     "outputAction": "Show Output Files",
     "outputFilesAria": "go to output files",
+    "popularFeaturedApps": "Popular Featured Apps",
```

Comment:
Any reason to call it `Popular Featured Apps` instead of just `Featured Apps` ?

Reply:
Also I guess for logged out view we can replaced `Public Apps` with `Featured Apps`